cumulusci/cli/cci.py

```diff
@@ -873,7 +880,35 @@ def org_info(config, org_name, print_json):
             )
         )
     else:
-        render_recursive(org_config.config)
+        UI_KEYS = [
+            "access_token",
```

Comment:
Sure, makes sense since we're wrapping it anyway. 

(Although, if it's a security concern, I'd note that `render_recursive` includes it and `refresh_token` for persistent orgs so we may want to filter those.)

Reply:
But I think this is the only use of render_recursive with an org config, and you're replacing it.